### PR TITLE
Add "Eliminate animation hitches with XCTest"

### PR DESCRIPTION
**Pull request by stherold**
## Checklist

- [x] I have read the [CONTRIBUTING.md](https://github.com/Blackjacx/WWDC/blob/master/CONTRIBUTING.md) file.
- [x] I have reviewed my own notes for typos, grammar and consistency (if you're submitting notes for a new session)

### CODE REVIEW

File: summaries/2020/10077.md [hunk 1]
```diff
@@ -2,6 +2,35 @@
 
 https://developer.apple.com/wwdc20/10077
 
-Presenters: _Example Guy, Another Person_
+Presenters: _Tanuja Mohan_
```
Comment (rogerluan) [inline summaries/2020/10077.md:5]:
```suggestion
Presenter: _Tanuja Mohan_
```

**Review by Blackjacx — APPROVED** [review]

File: summaries/2020/10077.md [hunk 2]
```diff
@@ -2,6 +2,35 @@
...
+- **Hitch ratio:** Hitch time in **ms** per second for a given duration
+- Apple doesn't use frames per second (fps) since:
+  - 60 or 120 fps is not always the desired target
+  - Display may intentionally not be updated
+  - Target frame rate may intentionally be lower than possible
+- Hitch ratio is always comparable across tests, following ratios are recommended:
+  - Good: < 5ms/s
+  - Warning: 5..10ms/s - users will start recognizing hitches
+  - Critical: >10ms/s
+- Can be measured using `XCTestMetrics`  and unit tests or for production apps using `MetricsKit` and Xcode Organizer
```
Comment (rogerluan) [inline summaries/2020/10077.md:19]:
```suggestion
- Can be measured using `XCTestMetrics` and unit tests or for production apps using `MetricsKit` and Xcode Organizer
```

**Review by rogerluan — COMMENTED** [review]
Very interesting concepts 👍

**Comment by Blackjacx** [comment]
@rogerluan you have a really good eye ;-) Fixed it on master: 23ae56a

**Comment by rogerluan** [comment]
Heh, cheers mate!

**Pull request merged**